examples/opt/model/test_text_gen.py

```diff
@@ -139,22 +151,23 @@ def inference_func(input_ids, past_key_values,
         inference_func_config = InferenceFuncConfig()
         for key in inference_func_config.__dataclass_fields__.keys():
             setattr(inference_func_config, key, getattr(raw_model.config, key))
+        inference_func_config.num_beams = num_beams
         print(inference_func_config)
 
     elif "alpa/opt" in model_name:
         alpa.init()
         num_pp_stages = max(2, alpa.get_global_cluster().num_hosts)
 
         name = model_name.split("-")[1].upper()
-        config = get_config(name, num_pp_stages=num_pp_stages)
+        config = get_config(name, num_pp_stages=num_pp_stages, batch_size=num_beams)
```

Comment:
`batch_size` = num_beams?

Reply:
If we need to support real batch size, the argument should be `batch_size=num_beams * batch_size`

Reply:
I think Yonghao's current way makes sense. `config.batch_size` is a feature of the model that defines the size of the batch dimension of the input, which would be `num_beams * batch_size` with beam search.